pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -299,15 +313,23 @@ func (r *RawIngressReconciler) Reconcile(isvc *v1beta1api.InferenceService) erro
 	if err != nil {
 		return err
 	}
+
+	// Check if existing service name has default suffix
+	useDefaultSuffix := false
+	existingService := &corev1.Service{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: constants.DefaultTransformerServiceName(isvc.Name), Namespace: isvc.Namespace}, existingService)
+	if err == nil {
+		useDefaultSuffix = true
+	}
```

Comment:
Wrap this logic in `getRawServiceHost` ? also transformer is optional component

Reply:
done